viewer/packages/camera-manager/src/DefaultCameraManager.ts

```diff
@@ -615,23 +615,14 @@ export class DefaultCameraManager implements CameraManager {
 
       if (wantNewScrollTarget && isZoomToCursor) {
         scrollStarted = true;
-        let newTarget: THREE.Vector3;
-
-        // Disable controls to prevent camera from moving while picking is happening.
-        // await is not working as expected because event itself is not awaited.
-        try {
-          this._controls.enabled = false;
-          const pointerEventData = {
-            offsetX: domElementRelativeOffset.offsetX,
-            offsetY: domElementRelativeOffset.offsetY,
-            button: e.button
-          };
-
-          newTarget = await this.calculateNewTarget(pointerEventData);
-        } finally {
-          this._controls.enabled = this.isEnabled;
-        }
 
+        const pointerEventData = {
```

Comment:
Controls disable part doesn't seem to have any effect now. If I remember correctly, it was added during development when calculation of new target was asynchronous.

Reply:
Ok... No really good way of for us to know though. I assume this part is not covered by unit tests?

Reply:
Ok... No really good way of for us to know though. I assume this part is not covered by unit tests?

Reply:
Yeah, it's not because it is more of a dynamic thing that also relies on picking.